app/index.html

```diff
@@ -13,6 +13,7 @@ <h1>Título</h1>
     <nav>
         <ul>
             <li><a href="pages/jean.html">Professor: Jean</a></li>
```

Comment:
```suggestion
            <li><a href="pages/jean.html">Professor: Jean</a></li>
```
Tem conflito na branch pois existem dois links no index, você tem que apagar a linha onde está o link do professor Jean no caso.